src/module/components/ngx-auth-firebaseui/auth.component.ts

```diff
@@ -163,6 +163,12 @@ export class AuthComponent implements OnInit, AfterViewInit, OnChanges, OnDestro
     if (changes.messageOnAuthSuccess || changes.messageOnAuthError) {
       this.updateAuthSnackbarMessages();
     }
+    if (changes.min) {
+      this.min = this.min != null ? Math.max(this.min, this.config.passwordMinLength) : this.config.passwordMinLength;
+    }
+    if (changes.max) {
+      this.max = this.max != null ? Math.min(this.max, this.config.passwordMaxLength) : this.config.passwordMaxLength;
+    }
```

Comment:
why did you choose the `nhOnChanges`'s hook ?
This will be not fired unless you change some inputs of the component.

Reply:
You are right. We need something in ngOnInit too to initialize min/max if they have not been setted.